src/schema/inputs/ProjectInput.ts

```diff
@@ -16,8 +16,8 @@ export class ProjectInput {
   @MaxLength(255)
   lifecyclePhase?: string;
 
-  @Field(() => [TagInput], { nullable: true })
-  tags!: TagInput[];
+  @Field(() => TagsetInput, { nullable: true })
+  tagset!: TagsetInput;
```

Comment:
tagset should be marked as nullable, not as required if the field is set to `nullable: true`